src/Service.Tests/Azure.DataApiBuilder.Service.Tests.csproj

```diff
@@ -5,6 +5,7 @@
     <IsPackable>false</IsPackable>
     <Nullable>disable</Nullable>
     <OutputPath>$(BaseOutputPath)\tests</OutputPath>
+    <NoWarn>NU1603</NoWarn>
   </PropertyGroup>
```

Comment:
Why does PR title say `suppress NU1603 for Private feed`? this is for our public feed right?